en/contributing-git.md

```diff
@@ -0,0 +1,345 @@
+Title: Working with Git and GitHub
+TODO:  Review commented out section: "Additional resources"
+table_of_contents: True
+
+
+# Working with Git and GitHub
+
+This page provides the essentials for working with Git and GitHub with the goal
+of contributing specifically to MAAS documentation. See the upstream projects
+for definitive documentation: [Git docs][git-docs] and
+[GitHub docs][github-docs].
+
+
+## Initial setup
+
+Several actions are required before you can begin. These only need to be done
+once.
+
+- GitHub account
+- User environment
+- Fork the MAAS docs repository on GitHub
+- Clone your MAAS docs fork locally
+- Add the upstream remote
+
+### GitHub account
+
+A [GitHub account][github-account] will be required. Sign up now!
```

Comment:
Sign up now! <- I know you're being friendly, but my own insecurities mean I see this as an order!